hammr/commands/scan/scan.py

```diff
@@ -179,6 +183,22 @@ def help_run(self):
         doParser = self.arg_run()
         doParser.print_help()
 
+    def check_overlay_option_is_allowed(self, name, overlay):
```

Comment:
thanks for this change, last thing, could be great that this method return a boolean instead on an int